docs/get-started/cost-billing/billing-process-for-harvard.md

```diff
@@ -1,23 +1,34 @@
 # Billing Process for Harvard University
 
-NERC billing at Harvard will be similar to how AWS accounts are billed and leverage
-the HUIT Central Billing [Portal](https://billing.huit.harvard.edu/). Your HUIT
+Direct Billing for NERC is a convenience service for Harvard Faculty and Departments.
+HUIT will pay the monthly invoices and then allocate the monthly usage costs on
+the Harvard University General Ledger. This follows a similar pattern with how
+other Public Cloud Providers (AWS, Azure, GCP) accounts are billed and leverage
+the [HUIT Central Billing Portal](https://billing.huit.harvard.edu/). Your HUIT
 Customer Code will be matched to your NERC Project Allocation Name as a Billing
 Asset. In this process you will be asked for your GL billing code, which you can
 change as needed per project. Please be cognizant that only a single billing code
 is allowed per billing asset. Therefore, if you have multiple projects with different
 funds, if you are able, please create a separate project for each fund. Otherwise,
 you will need to take care of this with internal journals inside of your department
-or lab. Each monthly billing cycle, the HUIT billing team will review and update
-the cost per asset. Subsequently, they will email billing cost information for
-each asset to the corresponding billing email addresses of all customers.
+or lab. During each monthly billing cycle, the NERC team will upload the billing
+Comma-separated values (CSV) files to the HUIT Central Billing system accessible
+AWS Object Storage (S3) bucket. The HUIT Central Billing system ingests billing
```

Comment:
Why are we using S3 when they already are using the sharedrive?

Reply:
HUIT Billing does not support the Shared Drive - they requires us to put the monthly CSVs for Harvard only to S3 bucket confirmed their parser to parse every day!

Reply:
MGHPCC & BUIT Billing have been using the sharedrive for invoicing for a long time. They are currently using it.

Reply:
Thank you for the in-person explanation. Now I understand that John G thought HUIT Billing was using the sharedrive and in fact, they were not.